operator/src/main/java/oracle/kubernetes/operator/helpers/CrdHelper.java

```diff
@@ -158,10 +158,14 @@ boolean isOutdatedBetaCrd(
 
     @Override
     public NextAction apply(Packet packet) {
-      if (context.version.isCrdV1Supported()) {
-        return doNext(context.verifyCrd(getNext()), packet);
-      } else {
-        return doNext(context.verifyBetaCrd(getNext()), packet);
+      try {
+        if (context.version.isCrdV1Supported()) {
+          return doNext(context.verifyCrd(getNext()), packet);
+        } else {
+          return doNext(context.verifyBetaCrd(getNext()), packet);
+        }
+      } catch (NullPointerException e) {
```

Comment:
Can we avoid catching and throwing NullPointerException here?

Reply:
Oops - thanks for catching that.  It was something I added to debug one of the test failures I found in the develop branch.